packages/frontend/core/src/blocksuite/presets/ai/entries/code-toolbar/setup-code-toolbar.ts

```diff
@@ -16,7 +16,10 @@ import type { AskAIButtonOptions } from '../../_common/components/ask-ai-button'
 import { buildAICodeItemGroups } from '../../_common/config';
 import { AIStarIcon } from '../../_common/icons';
 
-export function setupCodeToolbarEntry(codeToolbar: AffineCodeToolbarWidget) {
+export function setupCodeToolbarEntry(
```

Comment:
can we split this function to ai and non-ai ？

Reply:
Done